src/components/RepositoryDetail/RepositoryDetail.tsx

```diff
@@ -127,7 +128,7 @@ export const RepositoryHeadInfo: React.FunctionComponent<Props> = ({
           <meta name="og:title" content={info.title} />
           <meta name="og:url" content={info.pageUrl} />
           <meta name="og:image" content={info.imageUrl} />
-          <meta name="og:type" content="organization" />
+          <meta name="og:type" content="repository" />
```

Comment:
I am curious about this. Neither `organization` or `repository` is in the OGP types list: https://ogp.me/#types. Do we need to define a namespace that has them and prefix with that?

Reply:
Originally this had been copied from the organization page
https://github.com/datacite/akita/blob/8f5d72bf9a3b531dde34efff70b9e58f5df0107f/src/pages/ror.org/%5Brorid%5D.tsx#L280

I agree.  Neither organization, nor repository are part of the OPG vocabulary.

Reply:
I think this is worth further investigation but not necessarily a blocker for this work.

Reply:
Agreed :)